### PR TITLE
Migration to generic_workload

### PR DESCRIPTION
**Pull request by rsevilla87**
Signed-off-by: Raul Sevilla <rsevilla@redhat.com>

### Description
First approach of the issue described below

### Fixes
- https://github.com/cloud-bulldozer/benchmark-operator/issues/672

### CODE REVIEW

**Review by whitleykeith — APPROVED** [review]

**Review by jtaleric — APPROVED** [review]
LGTM

**Review by mohit-sheth — APPROVED** [review]

File: roles/api_load/tasks/main.yml [hunk 1]
```diff
@@ -1,20 +1,8 @@
 ---
-- block:
+- set_fact:
+    resources:
+    - "{{ role_path }}/templates/api_load.yml"
```
Comment (sjug) [inline roles/api_load/tasks/main.yml:4]: Do we want to have the resources declared in the `tasks` rather than in `defaults`?

**Review by sjug — COMMENTED** [review]
Looks fine overall, nit

**Pull request merged**